app/components/dialogs/prioritize-build-modal.js

```diff
@@ -0,0 +1,104 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import eventually from 'travis/utils/eventually';
+import { reads } from '@ember/object/computed';
+import { task } from 'ember-concurrency';
+import {
+  bindKeyboardShortcuts,
+  unbindKeyboardShortcuts
+} from 'ember-keyboard-shortcuts';
+
+export default Component.extend({
+  keyboardShortcuts: {
+    'esc': 'toggleConfirmationModal'
+  },
+  flashes: service(),
+
+  didInsertElement() {
+    this._super(...arguments);
+    bindKeyboardShortcuts(this);
+  },
+
+  willDestroyElement() {
+    this._super(...arguments);
+    unbindKeyboardShortcuts(this);
+  },
+  isOpen: false,
```

Comment:
This property is not used because template uses component attribute directly by `@isOpen` - is it expected?

Reply:
Removed now.